iron-overlay-behavior.html

```diff
@@ -15,7 +15,7 @@
 <link rel="import" href="iron-overlay-manager.html">
 
 <script>
-
+'use strict';
```

Comment:
If vulcanization or similar process would cause two scripts to be merged, it could end up like this:

```javascript
// Script 1:
(function() { /* etc ... */ })();
// Script 2:
'use strict';
// etc ...
```

In this case, the `use strict`; for "Script 2" would have no effect. If you wrap this script in an IIFE, then it becomes:

```javascript
// Script 1:
(function() { /* etc ... */ })();
// Script 2:
(function() {
  'use strict';
  // etc ...
})();
```

Then the `'use strict';` will always appear at the top of a function scope, so it will have an effect.